homeassistant/components/todoist/calendar.py

```diff
@@ -433,7 +433,7 @@ def __init__(
 
         self._coordinator = coordinator
         self._name = project_data[CONF_NAME]
-        # If no ID is defined, fetch all tasks.
+        # If no ID is defined, this is a custom project.
```

Comment:
Unrelated to the PR, but while investigating I wanted to make it clear what the value meant when it was `None`.